recipes/marslab/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "marslab" %}
+{% set version = "0.9.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/marslab-{{ version }}.tar.gz
+  sha256: 2b4bd12019b307f22e691f3f3743d17846c7da76d65592a15b5c2f210937c42a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - astropy
+    - clize
+    - cytoolz
+    - dustgoggles
+    - fs
+    - hypothesis
+    - matplotlib-base
+    - more-itertools
+    - numpy
+    - pandas
+    - pathos
+    - pdr
+    - python-dateutil
+    - python-Levenshtein
+    - scipy
+    - sympy
```

According to [upstream metadata](https://github.com/MillionConcepts/marslab/blob/v0.9.10/setup.py#L19-L28) these are the base requirements:

```suggestion
    - numpy
    - scipy
    - dustgoggles
    - fs
    - clize
    - pandas
    - more-itertools
    - pathos
    - cytoolz
```

You can add [some of the extras](https://github.com/MillionConcepts/marslab/blob/v0.9.10/setup.py#L30) but I think that adding the tests one are not necessary for the average user.

---

Apologies I did not realize I had left another test package in there. I've now removed hypothesis. We prefer to keep the extras in due to the proficiency level of our typical user.